### DISCUSSION TITLE
Yes, '+' is valid in email addresses

### DISCUSSION THREAD

**Discussion by slchorne** [community/community#38493] [Other Feature Feedback, Questions, & Ideas]
To all developers, every in the world : To anyone that is writing an email validation service : 

**The '+' sign is a valid character in an email address**

Just ask google. 

Please stop breaking login routines for people

(I couldn't think of anywhere else to broadcast this)

**Comment by david-engelmann** [comment]
Can confirm, I use a "+" in my email address

**Comment by SilasRodrigues19** [comment]
@slchorne I use this RegEx `/[^@ \t\r\n]+@[^@ \t\r\n]+\.([a-z\.]{1,3})\b/` 
Do you have anything better than that or a suggestion on how to improve?
